Packet++/src/LineBasedProtocol.cpp

```diff
@@ -0,0 +1,126 @@
+#include "Logger.h"
+#include "LineBasedProtocol.h"
+
+#include <string.h>
+
+namespace pcpp
+{
+
+    size_t LineBasedProtocolMessage::getOptionOffset() const
+	{
+		size_t maxLen;
+		if (m_DataLen < 5)
+			maxLen = m_DataLen;
+		else
+			maxLen = 5;
+
+		// Find <SP> if exists
+		uint8_t *pos = (uint8_t *)memchr(m_Data, 0x20, maxLen);
+		if (pos)
+			return pos - m_Data;
+
+		// Find Hyphen "-" if exists
+		pos = (uint8_t *)memchr(m_Data, 0x2d, maxLen);
+		if (pos)
+			return pos - m_Data;
+
+		return m_DataLen;
```

Comment:
if the command doesn't have arguments (i.e `FEAT\r\n`) this method will return the extra `\r\n`. I think we should omit that, no?

Reply:
I think it should return extra characters to indicate there is no option. But I added the fix to `LineBasedProtocolMessage::getCommandField()` to return the correct stripped string to the user.